cwl/src/main/scala/cwl/MyriadOutputTypeToWomType.scala

```diff
@@ -42,10 +42,15 @@ object MyriadOutputTypeToWomType extends Poly1{
       // If there's a single non null type, use that
       case (Array(), Array(singleNonNullType)) =>
         singleNonNullType.fold(MyriadOutputInnerTypeToWomType)
+      case (Array(), array: Array[MyriadOutputInnerType]) if array.size > 1  =>
+        val types = array.map(_.fold(MyriadOutputInnerTypeToWomType))
+        WomCoproductType(types.toList)
       // If there's a null type and a single non null type, it's a WomOptionalType
       case (Array(_), Array(singleNonNullType)) =>
         WomOptionalType(singleNonNullType.fold(MyriadOutputInnerTypeToWomType))
-      // Leave other "Coproduct types" unsupported for now
+      case (Array(_), array: Array[MyriadOutputInnerType]) if array.size > 1  =>
+        val types = array.map(_.fold(MyriadOutputInnerTypeToWomType))
+        WomOptionalType(WomCoproductType(types.toList))
```

Comment:
ToL: Code coverage tells me this is not being tested. If you've got the bandwidth It seems it would be a quick win to add a couple tests for these new `case`s and in `MyriadInputTypeToWomType` too.